.github/workflows/build.yml

```diff
@@ -104,8 +104,36 @@ jobs:
           message: |
             Build succeeded and deployed at http://${{ env.surge_url }} 
             (hash ${{ github.event.pull_request.head.sha }} deployed at ${{ steps.date.outputs.date }})
+  api_lint:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      ORG_LEVEL_TOKEN: ${{secrets.ORG_LEVEL_TOKEN}}
+    strategy:
```

Comment:
Since we are simply running formatting, we do not need to run on multiple python versions, let's just use 3.10